gitless/cli/gl_fetch.py

```diff
@@ -21,3 +22,5 @@ def parser(subparsers, _):
 
 def main(args, repo):
     core.git_wrap('fetch', *args.fetch_args)
+    pprint.ok('Fetching from upstream succeeded')
+    return True
```

Comment:
I would also suggest changing `return True` to something like  `return p.returncode == 0` where `p` is the object return by `git_wrap`. I need to see if this boolean is propagated upstream too.

Same applies to:
- [ ] gl_pull.py
- [ ] [gl_patch.py](https://github.com/goldstar611/gitless/pull/33/files#diff-e7155535a5cadd32eeb2bb38c5fcc31228792b6005b7b6c64fef2f2f29efd65d)